DOCS.md

```diff
@@ -188,6 +188,10 @@ vars
 : a map of variables to pass to the Terraform `plan` and `apply` commands.
 Each value is passed as a `-var <key>=<value>` option.
 
+var_files
+: a list of variable files to pass to the Terraform `plan` and `apply` commands.
```

Comment:
OK, I see what you are saying.  I got confused because `` and `` is being set in both places.  I wonder if those are redundant then.  I'll look into that separately.  Thanks!